src/components/network/NetworkView.tsx

```diff
@@ -172,6 +172,7 @@ const NetworkView: React.FC<RouteComponentProps<MatchParams>> = ({ match }) => {
       <Styled.PageHeader
         colors={theme.pageHeader}
         title={network.name}
+        description={'network.description'}
```

Comment:
This is causing a Typescript error because there is no `description` prop on the `PageHeader` component.